extensions/wikia/CreatePage/CreatePageHelper.class.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+class CreatePageHelper {
+
+	const ALLOWED_NAMESPACES = [ NS_MAIN ];
+	const FETCHED_TITLES_LIMIT = 6;
+	const FORBIDDEN_CHARACTERS_REGEX = '/[:\/]+/';
+
+	public static function getMostWantedPages() {
+		$WantedPagesPageResponse = ( new WantedPagesPage() )->doQuery();
+		$dbr = wfGetDB( DB_SLAVE );
+		$wantedPages = [];
+		$fetchedTitlesCount = 0;
+
+		while ( $row = $dbr->fetchObject( $WantedPagesPageResponse ) ) {
+			if ( $row->title &&
+				in_array( $row->namespace, static::ALLOWED_NAMESPACES ) &&
+				$fetchedTitlesCount < static::FETCHED_TITLES_LIMIT
+			) {
+				$wantedPageTitle = Title::newFromText( $row->title, $row->namespace );
```

Comment:
I think there is Title::newFromRow
https://www.mediawiki.org/wiki/Manual:Title.php